liveblog.php

```diff
@@ -245,7 +245,7 @@ public static function init() {
 		/**
 		 * This allows the users to filter their supported post types.
 		 */
-		self::$supported_post_types = apply_filters( 'modify_supported_post_types', $post_types );
+		self::$supported_post_types = apply_filters( 'liveblog_modify_supported_post_types', $post_types );
```

Comment:
Can you remove the `modify` from the filter? I think that would be better because it would almost look like the variable name we're already using: `liveblog_supported_post_types`